app/workers/geocode_application_address_worker.rb

```diff
@@ -0,0 +1,12 @@
+class GeocodeApplicationAddressWorker
+  include Sidekiq::Worker
+
+  def perform(application_form_id)
+    return unless Geocoder.config.api_key
+
+    application_form = ApplicationForm.find(application_form_id)
+    application_form.latitude, application_form.longitude = application_form.geocode
```

Comment:
If the quality of the address data varies it may be wise to restrict searches to GB (or UK?) otherwise you may end up getting coordinates for other countries

Reply:
Actually forget that - I guess you may have candidates that are living in other countries right?

Reply:
I don't think we can use this for international addresses though. They don't provide their address in a structured format.

![image](https://user-images.githubusercontent.com/42515961/100241570-bf07f100-2f2b-11eb-9283-d9c041c343cb.png)

Reply:
The worker doesn't get invoked for international addresses, and UK addresses don't surface country as an editable field (it defaults to GB). I think that's probably sufficient. WDYT?

Reply:
👍